backend/migrations/1676466055353-migrations.ts

```diff
@@ -9,12 +9,22 @@ export class migrations1676466055353 implements MigrationInterface {
     pass +
     `');`;
 
+  makeInsertMatchSQL = (p1: number, p2: number) =>
+    `INSERT INTO public.match(p1, p2, winner, powerup) VALUES(` +
+    p1 +
+    `,` +
+    p2 +
+    `, 0, false)`;
+
   public async up(queryRunner: QueryRunner): Promise<void> {
     const passHash = SHA256('password').toString();
     await queryRunner.query(this.makeInsertUserSQL('guest1', passHash));
     await queryRunner.query(this.makeInsertUserSQL('guest2', passHash));
     await queryRunner.query(this.makeInsertUserSQL('guest3', passHash));
     await queryRunner.query(this.makeInsertUserSQL('guest4', passHash));
+    await queryRunner.query(this.makeInsertMatchSQL(1, 2));
```

Comment:
UserID6までInsertするのであれば、事前にguest6まで作っておくと便利だと思います。
```suggestion
    await queryRunner.query(this.makeInsertUserSQL('guest4', passHash));
    await queryRunner.query(this.makeInsertUserSQL('guest5', passHash));
    await queryRunner.query(this.makeInsertUserSQL('guest6', passHash));
    await queryRunner.query(this.makeInsertMatchSQL(1, 2));
```

Reply:
そうなんですね！ごめんなさい、ユーザーとのつながりはまだ考えていなかったのですが、追加しました！